pkg/services/accesscontrol/roles.go

```diff
@@ -72,9 +82,11 @@ var (
 		},
 	}
 
-	usersOrgReadRole = RoleDTO{
-		Name:    usersOrgRead,
-		Version: 1,
+	orgUsersReaderRole = RoleDTO{
+		Name:        orgUsersReader,
+		DisplayName: "Organization users reader",
+		Description: "Read users in organization",
```

Comment:
```suggestion
		Description: "Read users in organization",
```
[an, the, your] organization